src/pages/login/loginPage.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from "../login/loginPage.module.css"
+
+
+function LoginPage({getter, setter, auth}) {
+    function Login() {
+        setter(!getter);
+
+        console.log("Knop wordt geklikt")
+    }
+
+
+    return (
+        <div className={styles.loginPage}>
+            <h1>Loginpagina</h1>
+
+            <p>{auth === true ? "hoi, je bent wel ingelogd" : "Je bent niet ingelogd"}</p>
+
+
+            <button
+                type="button"
+                onClick={Login}>
+                {auth === true ? "Uitloggen" : "Inloggen"}
+            </button>
+
+        </div>
```

Comment:
stuk stond wel in de opdracht, niet in de uitwerking, heb het zelf vrijwel gelijk gedaan dus wat mij betreft prima code

ter inzage nog ff screenshot van de uitwerkingen van de loginpagina

![image](https://user-images.githubusercontent.com/99805211/170492229-4c527856-27ca-44ed-9f47-bf8924e3266f.png)